GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -883,15 +889,20 @@ private void ForcePushBranchesCheckedChanged(object sender, EventArgs e)
         private void UpdateMultiBranchView()
         {
             _branchTable = new DataTable();
-            _branchTable.Columns.Add("Local", typeof(string));
-            _branchTable.Columns.Add("Remote", typeof(string));
-            _branchTable.Columns.Add("New", typeof(string));
-            _branchTable.Columns.Add("Push", typeof(bool));
-            _branchTable.Columns.Add("Force", typeof(bool));
-            _branchTable.Columns.Add("Delete", typeof(bool));
+            _branchTable.Columns.Add(LocalColumnName, typeof(string));
+            _branchTable.Columns.Add(RemoteColumnName, typeof(string));
+            _branchTable.Columns.Add(NewColumnName, typeof(string));
+            _branchTable.Columns.Add(PushColumnName, typeof(bool));
+            _branchTable.Columns.Add(ForceColumnName, typeof(bool));
+            _branchTable.Columns.Add(DeleteColumnName, typeof(bool));
             _branchTable.ColumnChanged += BranchTable_ColumnChanged;
-            var bs = new BindingSource { DataSource = _branchTable };
-            BranchGrid.DataSource = bs;
+            LocalColumn.DataPropertyName = LocalColumnName;
+            RemoteColumn.DataPropertyName = RemoteColumnName;
+            NewColumn.DataPropertyName = NewColumnName;
+            PushColumn.DataPropertyName = PushColumnName;
+            ForceColumn.DataPropertyName = ForceColumnName;
+            DeleteColumn.DataPropertyName = DeleteColumnName;
+            BranchGrid.DataSource = new BindingSource { DataSource = _branchTable };
```

Comment:
I think this is a good candidate for a future refactor to be replaced with a collection of strongly typed objects so we don't need to do things like these

Reply:
Same...